runtime/lua/vim/lsp.lua

```diff
@@ -89,10 +89,8 @@ end
 --- supported in any of the servers registered for the current buffer.
 ---@param method (string) name of the method
 function lsp._unsupported_method(method)
-  local msg = string.format(
-    'method %s is not supported by any of the servers registered for the current buffer',
-    method
-  )
+  local msg =
+    string.format('method %s is not supported by any of the servers registered for the current buffer', method)
```

Comment:
this is over 100 chars long. can we limit it?

Reply:
our C limit is 100 I believe, would prefer to match it for Lua

Reply:
Added a commit with 100 chars enforced for people's consideration